lib/ramble/ramble/language/modifier_language.py

```diff
@@ -81,8 +81,13 @@ def _execute_variable_modification(mod):
             raise DirectiveError('variable_modification directive given an invalid method.\n'
                                  f'  Valid methods are {str(supported_methods)}')
 
+        exp_modes = modes
+        if isinstance(exp_modes, list):
```

Comment:
Same comment about pushing the logic down.

Reply:
Done